port/unix/omrsysinfo.c

```diff
@@ -3268,8 +3271,9 @@ omrsysinfo_get_number_CPUs_by_type(struct OMRPortLibrary *portLibrary, uintptr_t
 static int32_t
 retrieveLinuxMemoryStatsFromProcFS(struct OMRPortLibrary *portLibrary, struct J9MemoryInfo *memInfo)
 {
-	int32_t rc = 0;
+	int32_t rc = 0, rc_swappiness = 0;
```

Comment:
- camel case for variable names: https://github.com/eclipse/omr/blob/master/doc/CodingStandard.md#variables
- one variable declaration per line: https://github.com/eclipse/omr/blob/master/doc/CodingStandard.md#variable-declarations
```suggestion
	int32_t rc = 0;
	int32_t rcSwappiness = 0;
```